shoop/simple_pricing/admin_form_part.py

```diff
@@ -38,8 +38,9 @@ def _build_fields(self):
             ((shop_id or 0, group_id or 0), price)
             for (shop_id, group_id, price)
             in SimpleProductPrice.objects.filter(product=self.product)
-            .values_list("shop_id", "group_id", "price")
+            .values_list("shop_id", "group_id", "price").exclude(group_id__isnull=True)
```

Comment:
This exclude is unneeded after you make group_id non-nullable in SimpleProductPrice.